package.json

```diff
@@ -57,9 +57,9 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "OASIS_GANACHE_COMPATIBILITY=1 node scripts/start.js",
     "build": "REACT_APP_ENV='prod' node scripts/build.js",
-    "build:test": "REACT_APP_ENV='test' node scripts/build.js",
+    "build:test": "OASIS_GANACHE_COMPATIBILITY=1 REACT_APP_ENV='test' node scripts/build.js",
```

Comment:
Change the `REACT_APP_ENV` to `OASIS_HIDE_MKR`. Do that in the `build` task as well.